main.py

```diff
@@ -1,7 +1,42 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
+    for name in genres:
+        Genre.objects.create(name=name)
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
```

Comment:
There is a typo in the actor's name 'Kianu Reaves'. It should be 'Keanu Reeves'. This is corrected later in the code, but it's better to avoid such typos initially.